concordia/settings_ecs.py

```diff
@@ -87,3 +87,6 @@
     "Transcribed and reviewed by volunteers participating in the "
     "By The People project at crowd.loc.gov."
 )
+
+if os.getenv("CONN_MAX_AGE_NONZERO"):
```

Comment:
Since we're not setting this to the actual value, what do you think about renaming this to something like `USE_PERSISTENT_DATABASE_CONNECTIONS`?